test/test-context.rb

```diff
@@ -132,6 +132,33 @@ def test_match_escalation_threshold
     assert_equal(-1, context.match_escalation_threshold)
   end
 
+  sub_test_case(".match_escalation?") do
```

Comment:
It should be `#force_match_escalation?` because this is a instance method not class method.